packages/contentful--create-contentful-app/src/template.ts

```diff
@@ -44,19 +53,19 @@ function validate(destination: string): void {
   const packageJSONLocation = `${destination}/package.json`;
   if (!existsSync(packageJSONLocation)) {
     throw new Error(
-      `Invalid template: missing "${packageJSONLocation}".`
```

Comment:
I would still call it template: the validation applies to examples (contentful things) and general templates